game/scripts/vscripts/Dialog.ts

```diff
@@ -88,6 +95,14 @@ class DialogController {
             Token: this.currentToken,
         });
 
+        this.currentParticleIndex = highlight(
+            {
+                type: "dialogCircle",
+                attach: true,
+                radius: 90,
+                units: [speaker]
+            })[0]
```

Comment:
Multiparticle highlights will break this, better not rely on it being a single one (ie. store the array and remove the entire array again)

Reply:
Change to accept an array and loop over it (though I don't see a reason why it should ever be a multiparticle call, but just in case)